requirements_test.txt

```diff
@@ -12,6 +12,7 @@ pytest-flakefinder~=1.1
 pytest-memray~=1.7; sys_platform != 'win32'
 pyfakefs~=5.7
 parameterized~=0.9.0
+Faker<35.0.0; python_version < '3.9'
```

Comment:
Also unsure about this one; at least it probably shouldn't be merged as part of this PR.